client/i18n/locales/espanol/motivation.json

```
@@ -115,6 +115,90 @@
-    }
+    },
+    {
+      "quote": "La victoria se consigue apelando a la inteligencia, nunca por pura fuerza.",
+      "author": "Ludwig von Mises"
+    },
+    {
+      "quote": "Seguir cuando crees que no puedes más, es lo que te hace diferente a los demás.",
+      "author": "Rocky Balboa"
+    },
+    {
+      "quote": "El hombre bien preparado para la lucha ya ha conseguido medio triunfo.",
+      "author": "Miguel de Cervantes Saavedra"
+    },
+    {
+      "quote": "Lo intentaste. Fracasaste. No importa. Inténtalo de nuevo. Fracasa otra vez. Fracasa mejor.",
+      "author": "Samuel Beckett"
+    },
+    {
+      "quote": "El mundo cambia con tu ejemplo no con tu opinión",
+      "author": "Paulo Coelho"
+    },
+    {
+      "quote": "Una persona que nunca cometió un error nunca intento nada nuevo.",
+      "author": "Albert Einstein"
+    },
+    {
+      "quote": "Queda prohibido no sonreír a los problemas, no luchar por lo que quieres, abandonarlo todo por miedo, no convertir en realidad tus sueños.",
+      "author": "Pablo Neruda"
+    },
+    {
+      "quote": "Cuando creíamos que teníamos todas las respuestas, de pronto, cambiaron todas las preguntas.",
+      "author": "Mario Benedetti"
+    },
+    {
+      "quote": "La gloria no consiste en no caer nunca, sino más bien en levantarse las veces que sea necesario.",
+      "author": "Mario Benedetti"
+    },
+    {
+      "quote": "Aprender a dudar es aprender a pensar.",
+      "author": "Octavio Paz"
+    },
+    {
+      "quote": "Enfrenta los obstáculos a medida que se presenten, no pierdas energía temiendo lo que pueda haber en el futuro.",
+      "author": "Isabel Allende"
+    },
+    {
+      "quote": "La vida es como un viaje sin meta. Lo que cuenta es el camino.",
+      "author": "Isabel Allende"
+    },
+    {
+      "quote": "La lógica te llevará desde A hasta B. La imaginación te llevará a cualquier parte",
+      "author": "Albert Einstein"
+    },
+    {
+      "quote": "No fracasé en la prueba. Encontré 100 formas de hacerla mal",
+      "author": "Benjamin Franklin"
+    },
+    {
+      "quote": "Si haces lo que siempre has hecho, llegarás donde siempre has llegado",
+      "author": "Tony Robbins"
+    },
+    {
+      "quote": "La mejor preparación para mañana es hacer lo mejor que puedas hoy",
+      "author": "H. Jackson Brown, Jr."
+    },
+    {
+      "quote": "El éxito es ir de fracaso en fracaso sin perder el entusiasmo",
+      "author": "Winston Churchill"
+    },
+    {
+      "quote": "No cuentes los días, haz que los días cuenten",
+      "author": "Muhammad Ali"
+    },
+    {
+      "quote": "La acción es la llave fundamental de todo éxito",
+      "author": "Pablo Picasso"
+    },
+    {
+      "quote": "La perfección no es alcanzable pero si la perseguimmos podemos conseguir la excelencia",
+      "author": "Vince Lombardi"
+    },
+    {
+      "quote": "La excelencia no es un acto, es un hábito",
+      "author": "Aristóteles"
+    },
```

```suggestion
    }
```
Remove trailing comma to fix linter error.

---

Thanks

---

@RafaelDavisH You need to commit @nhcarrigan's suggestion.  You have to click "Commit suggestion" to resolve it.